lib/server/express/updateExpressResponseFromNativeError.js

```diff
@@ -32,6 +33,11 @@ const updateExpressResponseFromNativeError = (response, error) => {
         title = 'Not found';
     }
 
+    if (error instanceof ConflictError) {
+        status = 409;
+        title = 'Conflict in data';
```

Comment:
Maybe a bit more explicit? `The request conflicts with existing data`?

Reply:
applied